.github/workflows/build.yml

```diff
@@ -101,26 +101,30 @@ jobs:
       - name: Generate code
         run: scripts/.generate-code.sh code
       - name: Build iOS (dev)
-        if: github.ref_name != 'production'
```

Comment:
How come all of these conditions have been changed to "if true" and "if false" ?

Reply:
Trying to force a production build and get it uploaded to Firebase or TestFlight/Google Play pre-release channel; this setting will be reverted

Reply:
Yes alright, totally fine 👍